lib/facet_name_converter.rb

```diff
@@ -48,4 +65,19 @@ def self.convert_to_model(model_name, column_name, property)
     # perform lookup, but fall back to provided column name if no match is found
     mappings[column_name.to_sym]&.dig(property.to_sym) || column_name
   end
+
+
+
+  # convert from Terra Interoperability Model to SCP metadata names 
+  #
+  # * *params*
+  #   - +column_name+ (String, Symbol) => facet name to convert
+  #   - +property+ (String, Symbol) => property to return (:id or :name only)
+  #
+  # * *returns*
+  #   - (String) => String value of requested column/property
+  def self.convert_to_scp(column_name, property)
```

Comment:
Rather than the new method here (which is totally fine, btw!), I would rather refactor `convert_to_model` so that we can go from any one convention to the other.  Essentially, you'd have `source_model` and `target_model` models (such as `:tim`, `:hca`, and `:scp`), and then depending on the combinations, you'd pick the right constant to convert to/from.

I'm thinking something like this (there might be a better way to assign `mappings`, this is quick & dirty):
```
def self.convert_to_model(source_model, target_model, column_name, property)
  if source_model.to_sym == :tim
    mappings = TIM_TO_ALEXANDRIA
  else
    mappings = target_model.to_sym == :hca ? ALEXANDRIA_TO_HCA : ALEXANDRIA_TO_TIM
  end
  # perform lookup, but fall back to provided column name if no match is found
  mappings[column_name.to_sym]&.dig(property.to_sym) || column_name
end
```